packages/x-charts/src/context/ChartApi.ts

```diff
@@ -10,7 +10,7 @@ export type PluginsPerSeriesType = {
   line: LineChartPluginsSignatures;
   scatter: ScatterChartPluginsSignatures;
   bar: BarChartPluginsSignatures;
-  pie: PieChartPluginSignatures;
+  pie: PieChartPluginsSignatures;
   /* Special value when creating a chart using composition. */
   composition: DefaultPluginSignatures;
```

Comment:
If all types end with `PluginsSignatures`, should we also rename `DefaultPluginSignatures` to `DefaultPluginsSignatures`?

To be honest, this feels weird to me. `PieChartPluginSignatures` feels more natural, even if it's less clear. Not sure if @mapache-salvaje has any opinion on this.